pkg/collector/adapters/config_to_probe_test.go

```diff
@@ -0,0 +1,243 @@
+package adapters_test
+
+import (
+	"github.com/open-telemetry/opentelemetry-operator/pkg/collector/adapters"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestSimpleCase(t *testing.T) {
+	configStr := `extensions:
```

Comment:
I think the tests can be converted to use a table-driven test. This would make it more readable.

Reply:
Good idea, thanks.